src/class/dfu/dfu_rt_device.h

```diff
@@ -29,36 +29,52 @@
 
 #include "common/tusb_common.h"
 #include "device/usbd.h"
+#include "dfu.h"
 
 #ifdef __cplusplus
  extern "C" {
 #endif
 
-
 //--------------------------------------------------------------------+
-// Common Definitions
+// Application Callback API (weak is optional)
 //--------------------------------------------------------------------+
+// Allow the application to update the status as required.
+// Is set to DFU_STATUS_OK during internal initialization and USB reset
+// Value is not checked to allow for custom statuses to be used
+void tud_dfu_runtime_set_status(dfu_mode_device_status_t status);
```

Comment:
what is your scenario to have the need to set runtime status. I am opted to remove if it is just-in-case scenario.

Reply:
This was purely for a just-in-case.  I will remove.

Reply:
Resolved in 2e2dc7b